src/panel_material_ui/widgets/button.py

```diff
@@ -33,7 +33,7 @@ class _ButtonLike(MaterialWidget):
         Delay (in milliseconds) to display the tooltip after the cursor has
         hovered over the Button, default is 500ms.""")
 
-    variant = param.Selector(objects=["contained", "outlined", "text"], default="contained", doc="""
+    variant = param.Selector(objects=["contained", "outlined", "text"], default="text", doc="""
```

Comment:
On purpose :-)

Reply:
I'll revert, we can discuss on proper defaults more. I'm open to aligning more closely with Mui, but in this case I think a button still has to resemble a button by default.